### PR TITLE
Schema, extension metadata, travis config and README

### PR DESCRIPTION
**Pull request by ColinMaudry**
Implementation of the following fields:

- tender/communication/atypicalToolUrl
- tender/communication/futureNoticeDate

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,26 @@
...
+In the European Union, this extension's fields correspond to [eForms BT-124 and BT-127](https://github.com/eForms/eForms). See [OCDS for the European Union](http://standard.open-contracting.org/profiles/eu/master/en/) for the correspondences to Tenders Electronic Daily (TED).
+
+## Example
+
+```json
+{
+  "tender": {
+    "atypicalToolUrl": "https://ecomm-procurement.example.net",
+    "futureNoticeDate": "2020-06-17T00:00:00+01:00"
+  }
```
Comment (jpmckinney) [inline README.md]: Missing a `communication` object around the two fields.

File: release-schema.json [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+        },
+        "futureNoticeDate": {
+          "title": "Future notice date",
+          "description": "The estimated date of the publication of the contract notice.",
+          "type": [
+            "string",
+            "null"
+          ],
+          "format": "date-time"
+        }
```
Comment (jpmckinney) [inline release-schema.json]: The mapping puts these inside a communication object.

File: release-schema.json [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+          "title": "Atypical tool URL",
+          "description": "Electronic communication with the procuring entity requires the use of tools and devices that are not generally available. These tools are accessible via this URL.",
+          "type": [
+            "string",
+            "null"
+          ]
+        },
+        "futureNoticeDate": {
+          "title": "Future notice date",
+          "description": "The estimated date of the publication of the contract notice.",
```
Comment (jpmckinney) [inline release-schema.json]: > The estimated date of publication of the next sequential notice.

File: release-schema.json [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+  "definitions": {
+    "Tender": {
+      "properties": {
+        "atypicalToolUrl": {
+          "title": "Atypical tool URL",
+          "description": "Electronic communication with the procuring entity requires the use of tools and devices that are not generally available. These tools are accessible via this URL.",
+          "type": [
+            "string",
+            "null"
+          ]
```
Comment (jpmckinney) [inline release-schema.json]: Add `"format": "uri"`

Comment (ColinMaudry) [reply release-schema.json]: Done

File: release-schema.json (hunk 3 above)
Comment (ColinMaudry) [reply release-schema.json]: Done

File: release-schema.json (hunk 2 above)
Comment (ColinMaudry) [reply release-schema.json]: Done

File: README.md (hunk 1 above)
Comment (ColinMaudry) [reply README.md]: Done

**Pull request merged**